database/seeds/UserSeeder.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class UserSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        factory(\App\Models\User::class, 10)->create();
```

Comment:
```suggestion
        factory(User::class, 10)->create();
```
use model lên trên nhé. Sửa các file khác

Reply:
e sửa lại rồi ạ